pages/index.js

```diff
@@ -5,7 +5,16 @@ import Postmate from 'postmate';
 
 const Home = () => {
 	const [paymentProps, setPaymentProps] = useState({
-		wallets: ['moneybutton', 'relayx']
+		wallets: ['vbox', 'moneybutton', 'relayx'],
+		outputs: [
```

Comment:
Can you remove this default on the outputs? I used to have defaults, but it caused some confusion sometimes when a user would see the default first then it would change to the correct amount and sometimes they'd try to swipe when they saw the cheaper default props

Reply:
Done